readthedocs/core/models.py

```diff
@@ -1,44 +1,46 @@
-"""Models for the core app."""
+"""
+Models for the core app.
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
 
-from __future__ import absolute_import
 import logging
 
 from django.db import models
 from django.utils.encoding import python_2_unicode_compatible
-from django.utils.translation import ugettext_lazy as _, ugettext
+from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import ugettext
+
 from annoying.fields import AutoOneToOneField
 
-STANDARD_EMAIL = "anonymous@readthedocs.org"
+STANDARD_EMAIL = 'anonymous@readthedocs.org'
 
 log = logging.getLogger(__name__)
 
 
 @python_2_unicode_compatible
-class UserProfile (models.Model):
-
-    """Additional information about a User."""
+class UserProfile(models.Model):
+    """
+    Additional information about a User.
+    """
```

Comment:
Without the `--make-summary-multi-line` it will be

    """Additional information about a User."""

Since, I removed that argument, it will be as you like.

> but modern suggestion is no space before docstring

What do you mean with that?